<!DOCTYPE html>
<!-- saved from url=(0040)https://function.princeton.edu/software/ -->
<html class="no-js"><!--<![endif]--><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
        
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <title>Software</title>
        <meta name="description" content="">
        <meta name="viewport" content="width=device-width, initial-scale=1">

        
        <link rel="stylesheet" type="text/css" href="./Software_files/functionlab.css">
	    <link rel="stylesheet" type="text/css" href="./Software_files/bootstrap.min.css">
 	    <link rel="stylesheet" type="text/css" href="./Software_files/bootstrap-theme.min.css">
        <link rel="shortcut icon" href="https://function.princeton.edu/static/img/favicon.ico">
        <link href="./Software_files/css" rel="stylesheet">

        
            
        

	
	<script async="" src="./Software_files/analytics.js.下载"></script><script>
	  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
	  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
	  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
	  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
	
	  ga('create', 'UA-90335537-1', 'auto');
	  ga('send', 'pageview');
	
	</script>
	
    <style>@font-face{font-family:"SANS-SERIF";src:local("SANS-SERIF");font-weight:400;}@font-face{font-family:"SANS-SERIF";font-weight:400;src:local("Microsoft YaHei UI"), local("Microsoft YaHei"), local("微软雅黑"), local("微軟雅黑"), local("WenQuanYi Micro Hei"), local("Droid Sans Fallback");unicode-range:U+2E80-9FFF, U+F900-FAFF, U+FE30-FE6F, U+FF00-FFEF, U+1B000-1B0FF, U+20000-2FA1F}@font-face{font-family:"SANS-SERIF";font-weight:bold;src:local("Microsoft YaHei UI"), local("Microsoft YaHei"), local("微软雅黑"), local("微軟雅黑"), local("WenQuanYi Micro Hei"), local("Droid Sans Fallback");unicode-range:U+2E80-9FFF, U+F900-FAFF, U+FE30-FE6F, U+FF00-FFEF, U+1B000-1B0FF, U+20000-2FA1F}@font-face{font-family:"RASA";src:local("RASA");font-weight:400;}@font-face{font-family:"RASA";font-weight:400;src:local("Microsoft YaHei UI"), local("Microsoft YaHei"), local("微软雅黑"), local("微軟雅黑"), local("WenQuanYi Micro Hei"), local("Droid Sans Fallback");unicode-range:U+2E80-9FFF, U+F900-FAFF, U+FE30-FE6F, U+FF00-FFEF, U+1B000-1B0FF, U+20000-2FA1F}@font-face{font-family:"RASA";font-weight:bold;src:local("Microsoft YaHei UI"), local("Microsoft YaHei"), local("微软雅黑"), local("微軟雅黑"), local("WenQuanYi Micro Hei"), local("Droid Sans Fallback");unicode-range:U+2E80-9FFF, U+F900-FAFF, U+FE30-FE6F, U+FF00-FFEF, U+1B000-1B0FF, U+20000-2FA1F}@font-face{font-family:"SERIF";src:local("SERIF");font-weight:400;}@font-face{font-family:"SERIF";font-weight:400;src:local("Microsoft YaHei UI"), local("Microsoft YaHei"), local("微软雅黑"), local("微軟雅黑"), local("WenQuanYi Micro Hei"), local("Droid Sans Fallback");unicode-range:U+2E80-9FFF, U+F900-FAFF, U+FE30-FE6F, U+FF00-FFEF, U+1B000-1B0FF, U+20000-2FA1F}@font-face{font-family:"SERIF";font-weight:bold;src:local("Microsoft YaHei UI"), local("Microsoft YaHei"), local("微软雅黑"), local("微軟雅黑"), local("WenQuanYi Micro Hei"), local("Droid Sans Fallback");unicode-range:U+2E80-9FFF, U+F900-FAFF, U+FE30-FE6F, U+FF00-FFEF, U+1B000-1B0FF, U+20000-2FA1F}@font-face{font-family:"HELVETICANEUE-LIGHT";src:local("HELVETICANEUE-LIGHT");font-weight:400;}@font-face{font-family:"HELVETICANEUE-LIGHT";font-weight:400;src:local("Microsoft YaHei UI"), local("Microsoft YaHei"), local("微软雅黑"), local("微軟雅黑"), local("WenQuanYi Micro Hei"), local("Droid Sans Fallback");unicode-range:U+2E80-9FFF, U+F900-FAFF, U+FE30-FE6F, U+FF00-FFEF, U+1B000-1B0FF, U+20000-2FA1F}@font-face{font-family:"HELVETICANEUE-LIGHT";font-weight:bold;src:local("Microsoft YaHei UI"), local("Microsoft YaHei"), local("微软雅黑"), local("微軟雅黑"), local("WenQuanYi Micro Hei"), local("Droid Sans Fallback");unicode-range:U+2E80-9FFF, U+F900-FAFF, U+FE30-FE6F, U+FF00-FFEF, U+1B000-1B0FF, U+20000-2FA1F}@font-face{font-family:"HELVETICA";src:local("HELVETICA");font-weight:400;}@font-face{font-family:"HELVETICA";font-weight:400;src:local("Microsoft YaHei UI"), local("Microsoft YaHei"), local("微软雅黑"), local("微軟雅黑"), local("WenQuanYi Micro Hei"), local("Droid Sans Fallback");unicode-range:U+2E80-9FFF, U+F900-FAFF, U+FE30-FE6F, U+FF00-FFEF, U+1B000-1B0FF, U+20000-2FA1F}@font-face{font-family:"HELVETICA";font-weight:bold;src:local("Microsoft YaHei UI"), local("Microsoft YaHei"), local("微软雅黑"), local("微軟雅黑"), local("WenQuanYi Micro Hei"), local("Droid Sans Fallback");unicode-range:U+2E80-9FFF, U+F900-FAFF, U+FE30-FE6F, U+FF00-FFEF, U+1B000-1B0FF, U+20000-2FA1F}@font-face{font-family:"RASA";src:local("RASA");font-weight:300;}@font-face{font-family:"RASA";font-weight:300;src:local("Microsoft YaHei UI"), local("Microsoft YaHei"), local("微软雅黑"), local("微軟雅黑"), local("WenQuanYi Micro Hei"), local("Droid Sans Fallback");unicode-range:U+2E80-9FFF, U+F900-FAFF, U+FE30-FE6F, U+FF00-FFEF, U+1B000-1B0FF, U+20000-2FA1F}@font-face{font-family:"RASA";font-weight:bold;src:local("Microsoft YaHei UI"), local("Microsoft YaHei"), local("微软雅黑"), local("微軟雅黑"), local("WenQuanYi Micro Hei"), local("Droid Sans Fallback");unicode-range:U+2E80-9FFF, U+F900-FAFF, U+FE30-FE6F, U+FF00-FFEF, U+1B000-1B0FF, U+20000-2FA1F}@font-face{font-family:"SERIF";src:local("SERIF");font-weight:300;}@font-face{font-family:"SERIF";font-weight:300;src:local("Microsoft YaHei UI"), local("Microsoft YaHei"), local("微软雅黑"), local("微軟雅黑"), local("WenQuanYi Micro Hei"), local("Droid Sans Fallback");unicode-range:U+2E80-9FFF, U+F900-FAFF, U+FE30-FE6F, U+FF00-FFEF, U+1B000-1B0FF, U+20000-2FA1F}@font-face{font-family:"SERIF";font-weight:bold;src:local("Microsoft YaHei UI"), local("Microsoft YaHei"), local("微软雅黑"), local("微軟雅黑"), local("WenQuanYi Micro Hei"), local("Droid Sans Fallback");unicode-range:U+2E80-9FFF, U+F900-FAFF, U+FE30-FE6F, U+FF00-FFEF, U+1B000-1B0FF, U+20000-2FA1F}@font-face{font-family:"INHERIT";src:local("INHERIT");font-weight:400;}@font-face{font-family:"INHERIT";font-weight:400;src:local("Microsoft YaHei UI"), local("Microsoft YaHei"), local("微软雅黑"), local("微軟雅黑"), local("WenQuanYi Micro Hei"), local("Droid Sans Fallback");unicode-range:U+2E80-9FFF, U+F900-FAFF, U+FE30-FE6F, U+FF00-FFEF, U+1B000-1B0FF, U+20000-2FA1F}@font-face{font-family:"INHERIT";font-weight:bold;src:local("Microsoft YaHei UI"), local("Microsoft YaHei"), local("微软雅黑"), local("微軟雅黑"), local("WenQuanYi Micro Hei"), local("Droid Sans Fallback");unicode-range:U+2E80-9FFF, U+F900-FAFF, U+FE30-FE6F, U+FF00-FFEF, U+1B000-1B0FF, U+20000-2FA1F}@font-face{font-family:"HELVETICA NEUE";src:local("HELVETICA NEUE");font-weight:400;}@font-face{font-family:"HELVETICA NEUE";font-weight:400;src:local("Microsoft YaHei UI"), local("Microsoft YaHei"), local("微软雅黑"), local("微軟雅黑"), local("WenQuanYi Micro Hei"), local("Droid Sans Fallback");unicode-range:U+2E80-9FFF, U+F900-FAFF, U+FE30-FE6F, U+FF00-FFEF, U+1B000-1B0FF, U+20000-2FA1F}@font-face{font-family:"HELVETICA NEUE";font-weight:bold;src:local("Microsoft YaHei UI"), local("Microsoft YaHei"), local("微软雅黑"), local("微軟雅黑"), local("WenQuanYi Micro Hei"), local("Droid Sans Fallback");unicode-range:U+2E80-9FFF, U+F900-FAFF, U+FE30-FE6F, U+FF00-FFEF, U+1B000-1B0FF, U+20000-2FA1F}@font-face{font-family:"ARIAL";src:local("ARIAL");font-weight:400;}@font-face{font-family:"ARIAL";font-weight:400;src:local("Microsoft YaHei UI"), local("Microsoft YaHei"), local("微软雅黑"), local("微軟雅黑"), local("WenQuanYi Micro Hei"), local("Droid Sans Fallback");unicode-range:U+2E80-9FFF, U+F900-FAFF, U+FE30-FE6F, U+FF00-FFEF, U+1B000-1B0FF, U+20000-2FA1F}@font-face{font-family:"ARIAL";font-weight:bold;src:local("Microsoft YaHei UI"), local("Microsoft YaHei"), local("微软雅黑"), local("微軟雅黑"), local("WenQuanYi Micro Hei"), local("Droid Sans Fallback");unicode-range:U+2E80-9FFF, U+F900-FAFF, U+FE30-FE6F, U+FF00-FFEF, U+1B000-1B0FF, U+20000-2FA1F}@font-face{font-family:"INHERIT";src:local("INHERIT");font-weight:500;}@font-face{font-family:"INHERIT";font-weight:500;src:local("Microsoft YaHei UI"), local("Microsoft YaHei"), local("微软雅黑"), local("微軟雅黑"), local("WenQuanYi Micro Hei"), local("Droid Sans Fallback");unicode-range:U+2E80-9FFF, U+F900-FAFF, U+FE30-FE6F, U+FF00-FFEF, U+1B000-1B0FF, U+20000-2FA1F}@font-face{font-family:"INHERIT";font-weight:bold;src:local("Microsoft YaHei UI"), local("Microsoft YaHei"), local("微软雅黑"), local("微軟雅黑"), local("WenQuanYi Micro Hei"), local("Droid Sans Fallback");unicode-range:U+2E80-9FFF, U+F900-FAFF, U+FE30-FE6F, U+FF00-FFEF, U+1B000-1B0FF, U+20000-2FA1F}@font-face{font-family:"A";src:local("A");font-weight:400;}@font-face{font-family:"A";font-weight:400;src:local("Microsoft YaHei UI"), local("Microsoft YaHei"), local("微软雅黑"), local("微軟雅黑"), local("WenQuanYi Micro Hei"), local("Droid Sans Fallback");unicode-range:U+2E80-9FFF, U+F900-FAFF, U+FE30-FE6F, U+FF00-FFEF, U+1B000-1B0FF, U+20000-2FA1F}@font-face{font-family:"A";font-weight:bold;src:local("Microsoft YaHei UI"), local("Microsoft YaHei"), local("微软雅黑"), local("微軟雅黑"), local("WenQuanYi Micro Hei"), local("Droid Sans Fallback");unicode-range:U+2E80-9FFF, U+F900-FAFF, U+FE30-FE6F, U+FF00-FFEF, U+1B000-1B0FF, U+20000-2FA1F}</style></head>

    <body class="template-listpage">
        
		  
        
<nav class="navbar navbar-custom navbar-static-top" role="navigation">
<div class="container">
	<div class="navbar-header">
		<button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
		<span class="sr-only">Toggle navigation</span>
		<span class="icon-bar"></span>
		<span class="icon-bar"></span>
		<span class="icon-bar"></span>
		</button>
		<a class="navbar-brand" href="https://function.princeton.edu/">
<!-- 			<img src="http://placehold.it/250x75&text=Logo" alt=""> -->
			<img src="./Software_files/logo.png">
			<div id="logoText" class="navbar-text">
				<!-- <h3 class="navbar-text">Troyanskaya Laboratory</h3> -->
				<h3>Troyanskaya Laboratory</h3>
				<p class="navbar-subtext">Bioinformatics and Functional Genomics</p>
			</div>
		</a>
	</div>
	<div class="collapse navbar-collapse navbar-right" id="bs-example-navbar-collapse-1">
		<ul class="nav navbar-nav">
	
	    <li class="">
	        <a href="https://function.princeton.edu/people/">people</a>
	    </li>       
	
	    <li class="">
	        <a href="https://function.princeton.edu/research/">research</a>
	    </li>       
	
	    <li class="active">
	        <a href="https://function.princeton.edu/software/">software</a>
	    </li>       
	
	    <li class="">
	        <a href="https://function.princeton.edu/publications/">publications</a>
	    </li>       
	
	    <li class="">
	        <a href="https://function.princeton.edu/join/">join</a>
	    </li>       
	
		</ul>
	</div>
</div>
</nav>

        <div class="main container">
        

	<div class="intro">

</div>
	
	<div class="container">
	
		
		 
	    <div class="current-software">
		
			<div class="row software-contain vcenter">
				
		    	
		    	<div class="col-lg-3 col-md-3 col-sm-3 text-center software-img-contain">
					<a href="http://asd.princeton.edu/" target="_blank" rel="noopener noreferrer"><img src="./Software_files/logo_asd.width-250.png" class="software-img"></a>
				</div>
				
				<div class="col-lg-8 col-md-8 col-sm-8 software-txt-contain orange-inv">
					<a href="http://asd.princeton.edu/" target="_blank" rel="noopener noreferrer">
			        	<h3 class="list-group-item-heading"><div class="rich-text"><p>ASD: Genome-wide predictions of Autism Spectrum Disorder-associated genes</p></div></h3>
			        </a>
			        <p class="list-group-item-text"></p><div class="rich-text"><p>Autism spectrum disorder (ASD) is a neurodevelopmental disorder characterized by deficits in social communication and restricted, repetitive patterns of behavior. ASD has a strong genetic basis but we still lack the full complement of autism-associated genes.&nbsp;</p><p>Here we present a genome-wide prediction of autism-associated genes based on known disease genes in the context of a human brain-specific gene interaction network.<br></p></div><p></p>
			    </div>
		    </div>	
		
			<div class="row software-contain vcenter">
				
		    	
		    	<div class="col-lg-3 col-md-3 col-sm-3 text-center software-img-contain">
					<a href="http://deepsea.princeton.edu/" target="_blank" rel="noopener noreferrer"><img src="./Software_files/logo_deepsea.width-250.jpg" class="software-img"></a>
				</div>
				
				<div class="col-lg-8 col-md-8 col-sm-8 software-txt-contain orange-inv">
					<a href="http://deepsea.princeton.edu/" target="_blank" rel="noopener noreferrer">
			        	<h3 class="list-group-item-heading"><div class="rich-text"><p>DeepSEA: Deep learning-based algorithmic framework for predicting chromatin effects</p></div></h3>
			        </a>
			        <p class="list-group-item-text"></p><div class="rich-text"><p>DeepSEA is a deep learning-based algorithmic framework for predicting the chromatin effects of sequence alterations with single nucleotide sensitivity. DeepSEA can accurately predict the epigenetic state of a sequence, including transcription factors binding, DNase I sensitivities, and histone marks in multiple cell types. It can be further utilized to predict the chromatin effects of sequence variants and prioritize regulatory variants.</p></div><p></p>
			    </div>
		    </div>	
		
			<div class="row software-contain vcenter">
				
		    	
		    	<div class="col-lg-3 col-md-3 col-sm-3 text-center software-img-contain">
					<a href="http://fntm.princeton.edu/" target="_blank" rel="noopener noreferrer"><img src="./Software_files/logo_fntm.width-250.png" class="software-img"></a>
				</div>
				
				<div class="col-lg-8 col-md-8 col-sm-8 software-txt-contain orange-inv">
					<a href="http://fntm.princeton.edu/" target="_blank" rel="noopener noreferrer">
			        	<h3 class="list-group-item-heading"><div class="rich-text"><p>FNTM: Functional Networks of Tissues in Mouse</p></div></h3>
			        </a>
			        <p class="list-group-item-text"></p><div class="rich-text"><p>Functional Networks of Tissues in Mouse (FNTM) provides biomedical researchers with tissue-specific predictions of functional relationships between proteins in the most widely used model organism for human disease, the laboratory mouse. Users can explore FNTM-predicted functional relationships for their tissues and genes of interest or examine gene function and interaction predictions across multiple tissues, all through an interactive, multi-tissue network browser. FNTM makes predictions based on integration of a variety of functional genomic data, including over 13,000 gene expression experiments, and prior knowledge of gene function. FNTM is an ideal starting point for clinical and translational researchers considering a mouse model for their disease of interest, researchers already working with mouse models who are interested in discovering new genes related to their pathways or phenotypes of interest, and biologists working with other organisms to explore the functional relationships of their genes of interest in specific mouse tissue contexts.</p></div><p></p>
			    </div>
		    </div>	
		
			<div class="row software-contain vcenter">
				
		    	
		    	<div class="col-lg-3 col-md-3 col-sm-3 text-center software-img-contain">
					<a href="http://giant.princeton.edu/" target="_blank" rel="noopener noreferrer"><img src="./Software_files/logo_giant_brain.width-250.jpg" class="software-img"></a>
				</div>
				
				<div class="col-lg-8 col-md-8 col-sm-8 software-txt-contain orange-inv">
					<a href="http://giant.princeton.edu/" target="_blank" rel="noopener noreferrer">
			        	<h3 class="list-group-item-heading"><div class="rich-text"><p>GIANT: Genome-scale Integrated Analysis of gene Networks in Tissues</p></div></h3>
			        </a>
			        <p class="list-group-item-text"></p><div class="rich-text"><p>We present genome-wide functional interaction networks for 144 human tissues and cell types developed using a data-driven Bayesian methodology that integrates thousands of diverse experiments spanning tissue and disease states. Tissue-specific networks predict lineage-specific responses to perturbation, identify the changing functional roles of genes across tissues and illuminate relationships among diseases. We introduce NetWAS, which combines genes with nominally significant genome-wide association study (GWAS) P values and tissue-specific networks to identify disease-gene associations more accurately than GWAS alone. Our webserver, GIANT, provides an interface to human tissue networks through multi-gene queries, network visualization, analysis tools including NetWAS and downloadable networks. GIANT enables systematic exploration of the landscape of interacting genes that shape specialized cellular functions across more than a hundred human tissues and cell types.</p></div><p></p>
			    </div>
		    </div>	
		
			<div class="row software-contain vcenter">
				
		    	
		    	<div class="col-lg-3 col-md-3 col-sm-3 text-center software-img-contain">
					<a href="http://imp.princeton.edu/" target="_blank" rel="noopener noreferrer"><img src="./Software_files/logo_imp.width-250.png" class="software-img"></a>
				</div>
				
				<div class="col-lg-8 col-md-8 col-sm-8 software-txt-contain orange-inv">
					<a href="http://imp.princeton.edu/" target="_blank" rel="noopener noreferrer">
			        	<h3 class="list-group-item-heading"><div class="rich-text"><p>IMP: Integrative Multi-species Prediction</p></div></h3>
			        </a>
			        <p class="list-group-item-text"></p><div class="rich-text"><p>IMP (Integrative Multi-species Prediction), originally released in 2012, is an interactive web server that enables molecular biologists to interpret experimental results and to generate hypotheses in the context of a large cross-organism compendium of functional predictions and networks. The system provides biologists with a framework to analyze their candidate gene sets in the context of functional networks, expanding or refining their sets using functional relationships predicted from integrated high-throughput data. IMP 2.0 integrates updated prior knowledge and data collections from the last three years in the seven supported organisms (<i>Homo sapiens</i>, <i>Mus musculus</i>, <i>Rattus norvegicus</i>, <i>Drosophila melanogaster</i>, <i>Danio rerio</i>, <i>Caenorhabditis elegans</i>, and <i>Saccharomyces cerevisiae</i>) and extends function prediction coverage to include human disease. IMP identifies homologs with conserved functional roles for disease knowledge transfer, allowing biologists to analyze disease contexts and predictions across all organisms. Additionally, IMP 2.0 implements a new flexible platform for experts to generate custom hypotheses about biological processes or diseases, making sophisticated data-driven methods easily accessible to researchers.<br></p></div><p></p>
			    </div>
		    </div>	
		
			<div class="row software-contain vcenter">
				
		    	
		    	<div class="col-lg-3 col-md-3 col-sm-3 text-center software-img-contain">
					<a href="http://nano.princeton.edu/" target="_blank" rel="noopener noreferrer"><img src="./Software_files/logo_nano.width-250.jpg" class="software-img"></a>
				</div>
				
				<div class="col-lg-8 col-md-8 col-sm-8 software-txt-contain orange-inv">
					<a href="http://nano.princeton.edu/" target="_blank" rel="noopener noreferrer">
			        	<h3 class="list-group-item-heading"><div class="rich-text"><p>In-silico nano-dissection</p></div></h3>
			        </a>
			        <p class="list-group-item-text"></p><div class="rich-text"><p>Cell-lineage-specific transcripts are essential for differentiated tissue function in metazoan organisms. They are frequently found to be the cause of hereditary disease and mediate progression of acquired diseases. Identifying the tissue specific transcriptome can guide disease gene identification in genetic studies and the development of organ specific therapeutic targets. This server performs an in silico nano-dissection, which is an approach we developed to identify genes with novel cell-lineage specific expression. This bioinformatics strategy leverages high-throughput functional genomics data from tissue homogenates to accurately predict genes enriched in specific cell types.</p></div><p></p>
			    </div>
		    </div>	
		
			<div class="row software-contain vcenter">
				
		    	
		    	<div class="col-lg-3 col-md-3 col-sm-3 text-center software-img-contain">
					<a href="http://libsleipnir.bitbucket.org/KNNImputer.html" target="_blank" rel="noopener noreferrer"><img src="./Software_files/logo_knn.width-250.jpg" class="software-img"></a>
				</div>
				
				<div class="col-lg-8 col-md-8 col-sm-8 software-txt-contain orange-inv">
					<a href="http://libsleipnir.bitbucket.org/KNNImputer.html" target="_blank" rel="noopener noreferrer">
			        	<h3 class="list-group-item-heading"><div class="rich-text"><p>KNNimpute: K-Nearest Neighbors Imputation</p></div></h3>
			        </a>
			        <p class="list-group-item-text"></p><div class="rich-text"><p>KNNimpute is an implementation of the k-nearest neighbors algorithm for estimation of missing values in microarray data. In our comparative study of several different methods used for missing value estimation we determined that KNNimpute provides superior performance in a variety of situations.</p></div><p></p>
			    </div>
		    </div>	
		
			<div class="row software-contain vcenter">
				
		    	
		    	<div class="col-lg-3 col-md-3 col-sm-3 text-center software-img-contain">
					<a href="http://seek.princeton.edu/" target="_blank" rel="noopener noreferrer"><img src="./Software_files/logo_seek.width-250.png" class="software-img"></a>
				</div>
				
				<div class="col-lg-8 col-md-8 col-sm-8 software-txt-contain orange-inv">
					<a href="http://seek.princeton.edu/" target="_blank" rel="noopener noreferrer">
			        	<h3 class="list-group-item-heading"><div class="rich-text"><p>SEEK: Search-based exploration of Expression compendia</p></div></h3>
			        </a>
			        <p class="list-group-item-text"></p><div class="rich-text"><p>SEEK is a computational gene co-expression search engine. SEEK provides biologists with a way to navigate the massive human expression compendium that now contains thousands of expression datasets. SEEK returns a robust ranking of co-expressed genes in the biological area of interest defined by the user's query genes. In the meantime, it also prioritizes thousands of expression datasets according to the user's query of interest. The unique strengths of SEEK include its support for multi-gene query and cross-platform analysis, as well as its rich visualization features.<br></p></div><p></p>
			    </div>
		    </div>	
		
			<div class="row software-contain vcenter">
				
		    	
		    	<div class="col-lg-3 col-md-3 col-sm-3 text-center software-img-contain">
					<a href="http://spell.yeastgenome.org/" target="_blank" rel="noopener noreferrer"><img src="./Software_files/logo_spell.width-250.jpg" class="software-img"></a>
				</div>
				
				<div class="col-lg-8 col-md-8 col-sm-8 software-txt-contain orange-inv">
					<a href="http://spell.yeastgenome.org/" target="_blank" rel="noopener noreferrer">
			        	<h3 class="list-group-item-heading"><div class="rich-text"><p>SPELL: Serial Pattern of Expression Levels Locator</p></div></h3>
			        </a>
			        <p class="list-group-item-text"></p><div class="rich-text"><p>SPELL is a query-driven search engine for large gene expression microarray compendia. Given a small set of query genes, SPELL identifies which datasets are most informative for these genes, then within those datasets additional genes are identified with expression profiles most similar to the query set. Both <a href="http://spell.yeastgenome.org/">SGD</a> and <a href="http://spell.caltech.edu:3000/">WormBase</a> now manage their own instances of SPELL.</p></div><p></p>
			    </div>
		    </div>	
		
			<div class="row software-contain vcenter">
				
		    	
		    	<div class="col-lg-3 col-md-3 col-sm-3 text-center software-img-contain">
					<a href="http://libsleipnir.bitbucket.org/index.html" target="_blank" rel="noopener noreferrer"><img src="./Software_files/logo_sleip.width-250.png" class="software-img"></a>
				</div>
				
				<div class="col-lg-8 col-md-8 col-sm-8 software-txt-contain orange-inv">
					<a href="http://libsleipnir.bitbucket.org/index.html" target="_blank" rel="noopener noreferrer">
			        	<h3 class="list-group-item-heading"><div class="rich-text"><p>Sleipnir: Library for computational functional genomics</p></div></h3>
			        </a>
			        <p class="list-group-item-text"></p><div class="rich-text"><p>Sleipnir is a C++ library enabling efficient analysis, integration, mining, and machine learning over genomic data. This includes a particular focus on microarrays, since they make up the bulk of available data for many organisms, but Sleipnir can also integrate a wide variety of other data types, from pairwise physical interactions to sequence similarity or shared transcription factor binding sites. All analysis is done with attention to speed and memory usage, enabling the integration of hundreds of datasets covering tens of thousands of genes. In addition to the core library, Sleipnir comes with a variety of pre-made tools, providing solutions to common data processing tasks and examples to help you use Sleipnir in your own programs. Sleipnir is free, open source, fully documented, and ready to be used by itself or as a component in your computational biology analyses.</p></div><p></p>
			    </div>
		    </div>	
		
			<div class="row software-contain vcenter">
				
		    	
		    	<div class="col-lg-3 col-md-3 col-sm-3 text-center software-img-contain">
					<a href="http://ursahd.princeton.edu/" target="_blank" rel="noopener noreferrer"><img src="./Software_files/logo_ursa.width-250.png" class="software-img"></a>
				</div>
				
				<div class="col-lg-8 col-md-8 col-sm-8 software-txt-contain orange-inv">
					<a href="http://ursahd.princeton.edu/" target="_blank" rel="noopener noreferrer">
			        	<h3 class="list-group-item-heading"><div class="rich-text"><p>URSAHD: Unveiling RNA Sample Annotation for Human Diseases</p></div></h3>
			        </a>
			        <p class="list-group-item-text"></p><div class="rich-text"><p>URSA (Unveiling RNA Sample Annotation), originally released in 2013,&nbsp;simultaneously estimated the probabilities that a given sample is associated with a particular tissue or cell-type. Individual cell-type models were constructed from more than ten thousand manually curated samples from GEO&nbsp;and then aggregated using Bayesian Correction.&nbsp;This method has been shown effective for both array-based and sequence-based genome-scale experiments.</p><p>Now, in addition to tissues and cell-types, URSAHD&nbsp;(Unveiling RNA Sample Annotation for Human Diseases) also measures hundreds of disease-specific signatures in a single gene expression profile. Each disease-specific model (i.e. gene weights) were computed based on thousands of clinical samples from GEO.</p></div><p></p>
			    </div>
		    </div>	
		
			<div class="row software-contain vcenter">
				
		    	
		    	<div class="col-lg-3 col-md-3 col-sm-3 text-center software-img-contain">
					<a href="http://yeti.princeton.edu/" target="_blank" rel="noopener noreferrer"><img src="./Software_files/Screenshot_from_2018-06-01_213349.width-250.png" class="software-img"></a>
				</div>
				
				<div class="col-lg-8 col-md-8 col-sm-8 software-txt-contain orange-inv">
					<a href="http://yeti.princeton.edu/" target="_blank" rel="noopener noreferrer">
			        	<h3 class="list-group-item-heading"><div class="rich-text"><p>YETI: Your Evidence Tailored Integration</p></div></h3>
			        </a>
			        <p class="list-group-item-text"></p><div class="rich-text"><p> Our method relies on a library of 237 Bayesian functional networks, 
each capturing the biology of a particular pathway or process. In 
aggregate, this functional library maps the entire human functional 
landscape. These genome-wide networks were constructed by integrating 
the growing public data compendium, currently including 35,300 
experiments. </p>
        <p> Based on the user's dataset, YETI selects relevant 
pathway/process networks by solving a global regularized linear 
regression problem. The resulting dataset-specific functional network 
represents an unbiased study of the user’s experiment that leverages the
 collection of public biological data. </p></div><p></p>
			    </div>
		    </div>	
		
		</div>
		
		
		
		<div class="old-software">
	 	<h4 class="collapse-head orange-inv" style="text-align:center;"><a data-toggle="collapse" href="https://function.princeton.edu/software/#collapse_old" class="" aria-expanded="true">Take a stroll through our software cemetery</a></h4>
	 	<div id="collapse_old" class="panel-collapse collapse in" aria-expanded="true" style="">
		
			<div class="row software-contain vcenter">
				
		    	
		    	<div class="col-lg-3 col-md-3 col-sm-3 text-center software-img-contain">
					<img src="./Software_files/logo_pilgrm.width-250.png" class="software-img">
				</div>
				
				<div class="col-lg-8 col-md-8 col-sm-8 software-txt-contain">
			        <h3 class="list-group-item-heading"><div class="rich-text"><p>PILGRM: The Platform for Interactive Learning by Genomics Results Mining</p></div></h3>
			        <p class="list-group-item-text"></p><div class="rich-text"><p>PILGRM is for the biologist with a set of proteins relevant to a disease, biological function or tissue of interest who wants to find additional players in that process. It uses a data driven method that provides added value for literature search results by mining compendia of publicly available gene expression datasets using lists of relevant and irrelevant genes (standards).</p></div><p></p>
			    </div>
		    </div>	
		
			<div class="row software-contain vcenter">
				
		    	
		    	<div class="col-lg-3 col-md-3 col-sm-3 text-center software-img-contain">
					<img src="./Software_files/logo_hefal.width-250.png" class="software-img">
				</div>
				
				<div class="col-lg-8 col-md-8 col-sm-8 software-txt-contain">
			        <h3 class="list-group-item-heading"><div class="rich-text"><p>HEFalMP: A Human Experimental/FunctionAL MaPper</p></div></h3>
			        <p class="list-group-item-text"></p><div class="rich-text"><p>A functional map is a way of usefully exploring information from thousands of experimental results, focused on a specific query of interest. This might mean finding data that pertains to a single gene/protein, a group of related (or unrelated) genes, a pathway, process, or set of disease-related genes. Functional maps rely on data integration to summarize genomic data as functional relationship networks. Each network encodes how likely it is for every pair of genes in the genome to interact functionally - possibly a direct interaction, like protein binding, or an indirect functional relationship, like participating in the same cellular process. Functional mapping analyzes portions of these networks related to user-specified groups of genes and biological processes and displays the results as probabilities (for individual genes), functional association p-values (for groups of genes), or graphically (as an interaction network). HEFalMp contains information from roughly 15,000 microarray conditions, over 15,000 publications on genetic and physical protein interactions, and several types of DNA and protein sequence analyses and allows the exploration of over 200 <i>H. sapeins</i>&nbsp;process-specific functional relationship networks, including a global, process-independent network capturing the most general functional relationships.</p></div><p></p>
			    </div>
		    </div>	
		
			<div class="row software-contain vcenter">
				
		    	
		    	<div class="col-lg-3 col-md-3 col-sm-3 text-center software-img-contain">
					<img src="./Software_files/logo_hidra.width-250.jpg" class="software-img">
				</div>
				
				<div class="col-lg-8 col-md-8 col-sm-8 software-txt-contain">
			        <h3 class="list-group-item-heading"><div class="rich-text"><p>HIDRA: Horizontally Integrated Dataset Relationship Analysis</p></div></h3>
			        <p class="list-group-item-text"></p><div class="rich-text"><p>HIDRA is a visualization and analyis framework for simultaneously exploring multiple microarray datasets at once. HIDRA allows users to quicky identify patterns common across many datasets as well as patterns unique to individual datasets. HIDRA is currently in beta testing and is still under development.</p></div><p></p>
			    </div>
		    </div>	
		
			<div class="row software-contain vcenter">
				
		    	
		    	<div class="col-lg-3 col-md-3 col-sm-3 text-center software-img-contain">
					<img src="./Software_files/logo_mefit.width-250.jpg" class="software-img">
				</div>
				
				<div class="col-lg-8 col-md-8 col-sm-8 software-txt-contain">
			        <h3 class="list-group-item-heading"><div class="rich-text"><p>MEFIT: Microarray Experiment Functional Integration Technology</p></div></h3>
			        <p class="list-group-item-text"></p><div class="rich-text"><p>MEFIT is a system for microarray integration. As a framework, MEFIT uses the results of many microarray experiments in combination with known biological process annotations (drawn from the Gene Ontology, KEGG, MIPS, or a biologist's own pathways of interest) to predict new gene pair functional relationships within the given biological functions. Or in other words, MEFIT is a system that takes microarray results and known functional annotations as inputs and produces predicted gene pair functional relationships as output.</p></div><p></p>
			    </div>
		    </div>	
		
			<div class="row software-contain vcenter">
				
		    	
		    	<div class="col-lg-3 col-md-3 col-sm-3 text-center software-img-contain">
					<img src="./Software_files/logo_grifn.width-250.jpg" class="software-img">
				</div>
				
				<div class="col-lg-8 col-md-8 col-sm-8 software-txt-contain">
			        <h3 class="list-group-item-heading"><div class="rich-text"><p>GRIFn: Gene Relationship Identification in Functional data</p></div></h3>
			        <p class="list-group-item-text"></p><div class="rich-text"><p>GRIFn is a system for evaluation of datasets and methods using a functional genomics gold standard based on curation by expert biolgists. It allows users to assess the ability of their datasets or methods to recapitulate known biology both in a global sense and in the context of specific biological processes. GRIFn allows enables fair comparisons between various data types and methods.</p></div><p></p>
			    </div>
		    </div>	
		
			<div class="row software-contain vcenter">
				
		    	
		    	<div class="col-lg-3 col-md-3 col-sm-3 text-center software-img-contain">
					<img src="./Software_files/logo_golem.width-250.jpg" class="software-img">
				</div>
				
				<div class="col-lg-8 col-md-8 col-sm-8 software-txt-contain">
			        <h3 class="list-group-item-heading"><div class="rich-text"><p>GOLEM: Gene Ontology Local Exploration Map</p></div></h3>
			        <p class="list-group-item-text"></p><div class="rich-text"><p>GOLEM is a tool for viewing, navigating, and analyzing the hierarchical structure and annotations to the gene ontology. The visualization component allows a user to see the local graph structure around a GO term of interest and navigate to nearby nodes. GOLEM also provides the ability to look for statistical enrichment of GO terms in lists of genes and then observe the relationships between those terms. GOLEM is available both as an applet for use online and as a standalone download.</p></div><p></p>
			    </div>
		    </div>	
		
			<div class="row software-contain vcenter">
				
		    	
		    	<div class="col-lg-3 col-md-3 col-sm-3 text-center software-img-contain">
					<img src="./Software_files/logo_biopixie.width-250.jpg" class="software-img">
				</div>
				
				<div class="col-lg-8 col-md-8 col-sm-8 software-txt-contain">
			        <h3 class="list-group-item-heading"><div class="rich-text"><p>bioPIXIE: Biological Pathway Inference from eXperimental Interaction Evidence</p></div></h3>
			        <p class="list-group-item-text"></p><div class="rich-text"><p>bioPIXIE is a novel system for biological data integration and visualization for S. cereviciae. It allows the user to discover interaction networks and pathways in which the user's gene(s) of interest participate. The system is based on a Bayesian algorithm for identification of biological networks based on integrated diverse genomic data.</p></div><p></p>
			    </div>
		    </div>	
		
			<div class="row software-contain vcenter">
				
		    	
		    	<div class="col-lg-3 col-md-3 col-sm-3 text-center software-img-contain">
					<img src="./Software_files/logo_charm.width-250.jpg" class="software-img">
				</div>
				
				<div class="col-lg-8 col-md-8 col-sm-8 software-txt-contain">
			        <h3 class="list-group-item-heading"><div class="rich-text"><p>ChARMview: Chromosomal Aberration Region Miner and Viewer</p></div></h3>
			        <p class="list-group-item-text"></p><div class="rich-text"><p>ChARMView is a visualization and analysis system for guided discovery of chromosomal abnormalities from microarray data. Our system facilitates manual or automated discovery of aneuploidies through dynamic visualization and integrated statistical analysis. ChARMView can be used with array CGH and gene expression microarray data, and multiple experiments can be viewed and analyzed simultaneously.</p></div><p></p>
			    </div>
		    </div>	
		
			<div class="row software-contain vcenter">
				
		    	
		    	<div class="col-lg-3 col-md-3 col-sm-3 text-center software-img-contain">
					<img src="./Software_files/logo_genevand.width-250.jpg" class="software-img">
				</div>
				
				<div class="col-lg-8 col-md-8 col-sm-8 software-txt-contain">
			        <h3 class="list-group-item-heading"><div class="rich-text"><p>GeneVAnD: Genomic Visualization and Analysis of Datasets</p></div></h3>
			        <p class="list-group-item-text"></p><div class="rich-text"><p>GeneVAnD is an implementation of several visualization techniques that incorporate meaningful statistics that are noise-robust for the purpose of analyzing the results of clustering algorithms on microarray data. This includes a rank-based visualization method that is more robust to noise, a difference display method to aid assessments of cluster quality and detection of outliers, and a projection of high dimensional data into a three dimensional space in order to examine relationships between clusters. Our methods are interactive and are dynamically linked together for comprehensive analysis. Further, our approach applies to both protein and gene expression microarrays, and our architecture is scalable for use on both desktop/laptop screens and large-scale display devices.</p></div><p></p>
			    </div>
		    </div>	
		
		</div>
	</div>

		
	
	</div>
    

        </div>

        <!-- FOOTER -->
        <hr class="footer-divider">
        <div class="container orange">
          <footer class="footer">
          	<div class="footer-left">© 2019 TroyanskayaLab &nbsp;|&nbsp; <a href="http://www.princeton.edu/" target="_blank" rel="noopener noreferrer">Princeton University</a></div>
            <div class="footer-right"><img class="footer-img" src="./Software_files/princeton_shield.original.jpg"></div>
          </footer>

        </div>
        
        <script type="text/javascript" src="./Software_files/functionlab.js.下载"></script>
        <script type="text/javascript" src="./Software_files/jquery-3.1.1.min.js.下载"></script>
        
        <script type="text/javascript" src="./Software_files/collapse.js.下载"></script>
        <script type="text/javascript" src="./Software_files/transition.js.下载"></script>

        
            
        
    

</body></html>